src/ast/semantic_analyser.cpp

```diff
@@ -836,6 +836,12 @@ void SemanticAnalyser::visit(Call &call)
             check_arg(call, Type::integer, 1, true);
           if (call.vargs->size() > 2)
             check_arg(call, Type::integer, 2, true);
+          if (map.type.IsStatsTy() && call.vargs->size() > 1)
+          {
+            warning("When printing stats maps, arguments top and div are "
```

Comment:
My gut feeling is this should be an error. Are there advantages to keeping it a warning?

Reply:
Actually I felt the same way at first, but I somehow felt it was too strong since it has always been allowed. But I guess error is more reasonable. Btw should this pr be included in CHANGELOG.md? I don't know if there is a section it belongs to.

Reply:
That's a good point about not breaking previous programs. Yeah let's keep it as a warning.

And yeah there should probably be a "Changed" entry